src/models/step_arguments/data_table.js

```diff
@@ -1,5 +1,30 @@
 import _ from 'lodash'
 
+const truths = ['true', 'TRUE', 'True', 'yes', 'YES', 'Yes', 'y', 'Y', '1']
+const trueFalse = (v) => !!~truths.indexOf(v)
+const asIs = (v) => v
```

Comment:
A function with this implementation is typically named `identity`. We can also just use lodash's `_.identity` here.